assets/js/components/adminbar/AdminBarClicks.js

```diff
@@ -55,27 +52,26 @@ const AdminBarClicks = ( { className } ) => {
 		dimensions: 'date',
 		url,
 	};
+
 	const searchConsoleData = useSelect( ( select ) => select( MODULES_SEARCH_CONSOLE ).getReport( reportArgs ) );
 	const hasFinishedResolution = useSelect( ( select ) => select( MODULES_SEARCH_CONSOLE ).hasFinishedResolution( 'getReport', [ reportArgs ] ) );
 	const error = useSelect( ( select ) => select( MODULES_SEARCH_CONSOLE ).getErrorForSelector( 'getReport', [ reportArgs ] ) );
+	const reportZero = isZeroReport( searchConsoleData );
+	// Store the WidgetReportZero component with useMemo to avoid maximum update depth exceeded error.
+	const zeroDataComponent = useMemo( () => <WidgetReportZero moduleSlug="search-console" widgetSlug={ WIDGET_SLUG } />, [ reportZero ] );
```

Comment:
Why is this dependent on the `reportZero` variable? The component should always render the same for the given props. Also, passing the `widgetSlug` here is not necessary, since that prop should already have been provided via `withWidgetComponentProps`.

Reply:
Yeah, I thought it had something to do with the max update error, but indeed it shouldn't depend on that variable.